docs/content/preview/drivers-orms/csharp/ysql.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Connect an application
+linkTitle: Connect an app
+description: C# driver for YSQL
+image: /images/section_icons/sample-data/s_s1-sampledata-3x.png
+menu:
+  preview:
+    identifier: csharp-driver-ysql
+    parent: csharp-drivers
+    weight: 400
+type: docs
+---
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+  <li>
+    <a href="../postgres-npgsql/" class="nav-link active">
+      YSQL
+    </a>
+  </li>
+  <li>
+    <a href="../ycql/" class="nav-link">
+      YCQL
+    </a>
+  </li>
+</ul>
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+
+  <li >
+    <a href="../ysql/" class="nav-link active">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      YugabyteDB C# Smart Driver
+    </a>
+  </li>
+
+  <li >
+    <a href="../postgres-npgsql/" class="nav-link">
+      <i class="icon-postgres" aria-hidden="true"></i>
+      PostgreSQL Npgsql Driver
+    </a>
+  </li>
+
+</ul>
+
+The [Yugabyte Npgsql smart driver](https://github.com/yugabyte/npgsql) is a distributed .NET driver for [YSQL](../../../api/ysql/) built on the [PostgreSQL Npgsql driver](https://github.com/npgsql/npgsql/tree/main/src/Npgsql), with additional [connection load balancing](../../smart-drivers/) features.
+
+{{< note title="YugabyteDB Managed" >}}
+
+To use smart driver load balancing features when connecting to clusters in YugabyteDB Managed, applications must be deployed in a VPC that has been peered with the cluster VPC. For applications that access the cluster from a non-peered network, use the upstream PostgreSQL driver instead; in this case, the cluster performs the load balancing. Applications that use smart drivers from non-peered networks fall back to the upstream driver behaviour automatically. For more information, refer to [Using smart drivers with YugabyteDB Managed](../../smart-drivers/#using-smart-drivers-with-yugabytedb-managed).
```

Comment:
@Sfurti-yb - Can you clarify if anything else needs to be added for YBM ?

Reply:
No, this seems fine.